app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php

```diff
@@ -2,207 +2,367 @@
+                                                                src="{{ Vite::asset('resources/images/canyon-ai-headshot.jpg') }}"
+                                                                alt="AI Assistant avatar"
+                                                            >
+                                                        </div>
+                                                    </div>
+                                                    <div
+                                                        class="agent-turn relative flex w-[calc(100%-50px)] flex-col gap-1 md:gap-3 lg:w-[calc(100%-115px)]">
+                                                        <div class="flex max-w-full flex-grow flex-col gap-3">
+                                                            <div
+                                                                class="flex min-h-[20px] flex-col items-start gap-3 overflow-x-auto break-words">
+                                                                <div class="prose dark:prose-invert">
+                                                                    {!! str($message->message)->markdown()->sanitizeHtml() !!}
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                        <div class="flex justify-between empty:hidden lg:block">
+                                                            {{--                                                            <div --}}
+                                                            {{--                                                                class="visible mt-2 flex justify-center gap-2 self-end text-gray-400 md:gap-3 lg:absolute lg:right-0 lg:top-0 lg:mt-0 lg:translate-x-full lg:gap-1 lg:self-center lg:pl-2"> --}}
+                                                            {{--                                                                <x-filament::icon --}}
+                                                            {{--                                                                    class="ml-auto flex h-6 w-6 cursor-pointer items-center gap-2 rounded-md p-1 text-xs hover:bg-gray-100 hover:text-gray-700 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-200 disabled:dark:hover:text-gray-400" --}}
+                                                            {{--                                                                    icon="heroicon-o-clipboard" --}}
+                                                            {{--                                                                /> --}}
+                                                            {{--                                                            </div> --}}
```

Comment:
Do we need this anymore?

Reply:
Leaving it here for now just in case Joe wants this copy/paste functionality that ChatGPT has.